src/vs/languages/html/common/htmlTags.ts

```diff
@@ -319,71 +319,71 @@ export const HTML_TAGS: ITagSet = {
 };
 
 export const IONIC_TAGS: ITagSet = {
-	'ion-checkbox': new HTMLTagSpecification('',
+	'ion-checkbox': new HTMLTagSpecification(nls.localize('tags.ion.checkbox', 'The checkbox is no different than the HTML checkbox input, except it\'s styled differently. The checkbox behaves like any AngularJS checkbox.'),
```

Comment:
I've used nls.localize, with a string ID for each label; is there anything else to do for localization?